backend/src/main/java/com/jobwait/persistence/PersistenceUtil.java

```diff
@@ -5,20 +5,20 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import com.jobwait.persistence.adapters.AdapterException;
+import com.jobwait.fault.FaultException;
 import com.jobwait.persistence.adapters.PostgresAdapter;
 
 public class PersistenceUtil {
```

Comment:
Note that [PersistenceUtil](https://github.com/pecasdev/jobwait/pull/18/files#diff-327331ccff3bea53f221fb0d709b2d2a77f1d3db5aa70c8025aa798c75a19b86) still throws SqlExceptions instead of FaultExceptions. I was on the fence about it throwing FaultException because we don't really want to tell the user that we got a single row instead of 2, they don't care or need to know that, they just need to know that there was an error with getting the user. I think keeping SQLExceptions in the persistence layer makes sense cause persistence is SQL, but idk, I'm on the fence

Reply:
yeah I can work with that logic, sweet